javaee/api/src/main/java/org/jboss/forge/addon/javaee/security/AuthenticationMethod.java

```diff
@@ -0,0 +1,18 @@
+package org.jboss.forge.addon.javaee.security;
+
+/**
+ * The value of the auth-method element in the web.xml.
+ * <p>
+ * As defined in the Servlet specification, can be BASIC, DIGEST, FORM and CLIENT-CERT.
+ *
+ * @author <a href="mailto:ivan.st.ivanov@gmail.com">Ivan St. Ivanov</a>
+ */
+public enum AuthenticationMethod
```

Comment:
Authentication method can be any value. These values should be available as a hint (Auto-complete) to the user.